src/index.ts

```diff
@@ -32,7 +35,7 @@ app.use(
 addRequestSizeMiddleware(app);
 addSwaggerRoutes(app);
 
-logger.info('Using new routes');
+logger.error('Using new routes');
```

Comment:
why error ?

Reply:
This was added for debugging purpose. Will change this

Reply:
updated